src/pages/home/report/ReportActionCompose.js

```diff
@@ -162,6 +174,43 @@ class ReportActionCompose extends React.Component {
         }
     }
 
+    /**
+     * Save the location of a native press event.
+     *
+     * @param {Object} nativeEvent
+     */
+    capturePressLocation(nativeEvent) {
```

Comment:
This function seems like it's only used once, so I think it would be better to just do this inline in `showEmojiPicker`